### DISCUSSION TITLE
Development

### DISCUSSION THREAD

**Discussion by Expalmi** [vwt12eh8/hassio-ecoflow#96] [General]
Hello,

thank you for your work.

Do you have time or motivation to further develop this integration, if not, this will be ok.

Please Inform the users of your integration, so they can decide to use or not to use it.

Andreas

**Comment by MatGrump** [comment]
I guess the silence speaks volumes. Someone should fork and take over.

**Comment by Ne0-Hack3r** [comment]
@vwt12eh8 

Do you have any plans to continue supporting this HACS integration?
If not, do you have anyone in mind that might be willing to fork it and continue supporting it?
